tests/crowdsourcing/tasks/model_chat/test_model_chat_analysis.py

```diff
@@ -23,6 +26,42 @@
     )
     from parlai.crowdsourcing.utils.tests import check_stdout
 
+    class TestModelChatResultsCompiler(ModelChatResultsCompiler):
+        def get_task_data(self) -> List[Dict[str, Any]]:
+            fake_jsons = []
+            read_folders = []
+            date_strings = []
```

Comment:
super minor, but it'd be cleaner to skip the initialization of these here - date_strings is re-initialized below, and `read_folders` is effectively set to be equal to `folders`